server/runtime/src/main/java/org/infinispan/server/configuration/hotrod/HotRodServerConfigurationParser.java

```diff
@@ -252,10 +252,11 @@ private void parseTopologyStateTransfer(ConfigurationReader reader, HotRodServer
                break;
             }
             case LAZY_RETRIEVAL: {
-               if (reader.getSchema().since(11, 0))
```

Comment:
Isn't the logic wrong here? If `since(16,0)` it should fail with the unexpected attribute, otherwise log a warning.